### DISCUSSION TITLE
ResteasyClient stop working after KC 18 update

### DISCUSSION THREAD

**Discussion by AndreiMihaiRad** [keycloak/keycloak#11810] [Archive]
The actual error is 
`
ERROR: Failed to start server in (production) mode
ERROR: java.lang.reflect.InvocationTargetException
ERROR: org.jboss.resteasy.client.jaxrs.spi.ClientConfigProvider: org.wildfly.security.auth.client.spi.RESTEasyClientConfigProviderImpl Unable to get public no-arg constructor
ERROR: org/wildfly/client/config/ConfigXMLParseException
ERROR: org.wildfly.client.config.ConfigXMLParseException
`

Scenario: I'm using "org.jboss.resteasy:resteasy-client:4.7.5.Final" in my custom module which was imported under /providers folder along with following jars (resteasy-client-4.7.5.Final.jar and (resteasy-client-api-4.7.5.Final.jar)

No code change was made from KC 17, where our module was working fine.

I checked to see if there is some version mismatch between imported jars and the jars that come under keycloak distribution but I don't find any clue. Please Help. 
Any suggestion would be welcome, Thank you.

**Comment by kkcmadhu** [comment]
check if this helps https://github.com/keycloak/keycloak/discussions/10729
